source/backend/dal/Repositories/LeaseRepository.cs

```diff
@@ -1071,10 +1071,16 @@ private static ExpressionStarter<PimsLease> GenerateCommonLeaseQuery(LeaseFilter
                  || l.PimsLeaseStakeholders.Any(stakeholder => stakeholder.Organization != null && EF.Functions.Like(stakeholder.Organization.OrganizationName, $"%{filter.TenantName}%")));
             }
 
-            if (!string.IsNullOrWhiteSpace(filter.PinOrPid))
+            if (!string.IsNullOrWhiteSpace(filter.Pid))
             {
-                var pinOrPidValue = filter.PinOrPid.Replace("-", string.Empty).Trim().TrimStart('0');
-                predicateBuilder = predicateBuilder.And(l => l.PimsPropertyLeases.Any(pl => pl != null && (EF.Functions.Like(pl.Property.Pid.ToString(), $"%{pinOrPidValue}%") || EF.Functions.Like(pl.Property.Pin.ToString(), $"%{pinOrPidValue}%"))));
+                var pidValue = filter.Pid.Replace("-", string.Empty).Trim().TrimStart('0');
+                predicateBuilder = predicateBuilder.And(l => l.PimsPropertyLeases.Any(pl => pl != null && EF.Functions.Like(pl.Property.Pid.ToString(), $"%{pidValue}%")));
+            }
+
+            if (!string.IsNullOrWhiteSpace(filter.Pin))
+            {
+                var pinValue = filter.Pin.Replace("-", string.Empty).Trim().TrimStart('0');
```

Comment:
A pins shouldn't have any hyphens, was there a reason for this?

Reply:
I wasn't sure about the PIN format but this is how it was handled before.